fired_guys.py

```diff
@@ -1,9 +1,19 @@
+import argparse
 import os
-import json
 import requests
 import datetime
 
-def run(count=5):
+
+def run(default_count=5):
+    parser = argparse.ArgumentParser()
```

Comment:
I think it might make more sense to put the argument parsing stuff under `if __name__ == '__main__':` and pass into `run()`, so `run()` could technically still be used outside the context of running this as a script

Reply:
I tried to, but for some reason argparse doesnt get anything, i know... weird